Update multi-factor-authentication.adoc introduction (#100)
mulesoft/docs-access-management: modules/ROOT/pages/multi-factor-authentication.adoc

include::_attributes.adoc[]
endif::[]

Multi-factor authentication provides an additional layer of verification for Anypoint Platform users.

* As an organization administrator, you can require all non-SSO users in your organization to use multiple verification methods to successfully log in.

> * An an organization administrator, you have the option to exempt specific user accounts from the multi-factor authentication requirement, in order to allow existing integrations to continue operating without interruption. 

Reviewer: ```suggestion
* As an organization administrator, you can exempt specific user accounts from the multi-factor authentication requirement in order to allow existing integrations to continue operating without interruption. 
```
